content/vendor/native-helm/helm-charts-install-ordering.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2021-10-26
+linktitle: "Native Helm Charts Install Ordering"
+title: Native Helm Charts Install Ordering
+weight: 70010
+isBeta: true
+---
+
+When deploying applications with native Helm charts, your charts are deployed to the cluster in parallel with any other kubernetes yaml within your app.
+This includes CRDs, so any custom resources referenced within a native Helm chart must themselves be created by a native Helm chart, and used only within that native Helm chart.
```

Comment:
Maybe reword this sentence to something like:

This includes custom resource definitions (CRDs). So any custom resources that are referenced in a native Helm chart must also be created by a native Helm chart, and used only in that native Helm chart.